test/argument-parsing.coffee

```diff
@@ -1,3 +1,6 @@
+path = require 'path'
```

Comment:
You need to start the file with `return unless require?` so that this doesn’t break the browser tests `cake test:browser`